docs/ext/commands/slash_commands.rst

```diff
@@ -383,6 +383,23 @@ For example:
     ):
         ...
 
+
+In case you need don't want to use Param or need to use self in a cog you may
+create autocomplete options with the decorator @slash_command.autocomplete()
```

Comment:
```suggestion
In case you need don't want to use :class:`Param <ext.commands.Param>` or need to use ``self`` in a cog you may
create autocomplete options with the :func:`autocomplete <ext.commands.InvokableSlashCommand.autocomplete>` decorator:
```